format/format.go

```diff
@@ -853,10 +853,26 @@ func (w *writer) listWriter() entryWriter {
 // groupIterable will group the `elements` slice into slices according to their
 // location: anything on the same line will be put into a slice.
 func groupIterable(elements []interface{}, last *ast.Location) [][]interface{} {
+	generated := false
+	for _, e := range elements {
+		if t, ok := e.(*ast.Term); ok {
+			if v, ok := t.Value.(ast.Var); ok {
```

Comment:
Should we use a visitor here? I'm not sure if we can end up with cases like `f([__local0__])`